src/doc_data.cpp

```diff
@@ -126,33 +140,33 @@ void LayerDataHandler::convert(
         }
       }
     } else
-    if (args_->trainMode == 2) {
-      // pick one random rhs as lhs, the rest becomes rhs features
-      auto idx = rand() % example.RHSFeatures.size();
-      for (int i = 0; i < example.RHSFeatures.size(); i++) {
-        if (i == idx) {
-          insert(rslt.LHSTokens, example.RHSFeatures[i], args_->dropoutLHS);
-        } else {
-          insert(rslt.RHSTokens, example.RHSFeatures[i], args_->dropoutRHS);
+      if (args_->trainMode == 2) {
```

Comment:
I tink we don't need to change indentation here as it's consistent across files.

Reply:
fixed